tests/MongoDB.Driver.Tests/AggregateFluentTests.cs

```diff
@@ -196,6 +196,71 @@ public void ChangeStream_should_add_the_expected_stage_when_options_is_null()
             result.Should().Be(1);
         }
 
+        [SkippableFact]
+        public void Function_should_return_expected_result()
+        {
+            RequireServer.Check().Supports(Feature.AggregateFunction);
+
+            var client = CreateClient();
+            var databaseName = "test";
+            var collectionName = "collection";
+            DropCollection(client, databaseName, collectionName);
+            var collection = client.GetDatabase(databaseName).GetCollection<BsonDocument>(collectionName);
+            var players = new[]
+            {
+                BsonDocument.Parse("{ _id : 1, name : 'Miss Cheevous',  scores : [10, 5, 10] }"),
+                BsonDocument.Parse("{ _id : 2, name : 'Miss Ann Thrope', scores : [10, 10, 10] }"),
+                BsonDocument.Parse("{ _id : 3, name : 'Mrs. Eppie Delta', scores : [9, 8, 8] }")
+            };
```

Comment:
Nit: use spaces after opening braces and before closing braces (including array braces):

    var players = new[]
    {
        BsonDocument.Parse("{ _id : 1, name : 'Miss Cheevous',  scores : [ 10, 5, 10 ] }"),
        BsonDocument.Parse("{ _id : 2, name : 'Miss Ann Thrope', scores : [ 10, 10, 10 ] }"),
        BsonDocument.Parse("{ _id : 3, name : 'Mrs. Eppie Delta', scores : [ 9, 8, 8 ] }")
    };

Reply:
Note; I've added this convention to our evolving style guide

Reply:
As I remember, in the previous similar dispute we decided that array braces should be without opening/ending spaces :) But ok, done

Reply:
I searched our code base and we are 50/50 on this one.